gradle/wrapper/gradle-wrapper.properties

```diff
@@ -15,6 +15,6 @@
 #
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.4-all.zip
```

Comment:
anything stopping us from going to gradle 8?

Reply:
Some unknowns are there, basically compatibility with AGP as well as some internal APIs are using. Will target it separately.